releasenotes/notes/add-hf-initial-point-ac29baf3091fa5b1.yaml

```diff
@@ -0,0 +1,51 @@
+---
+features:
+  - |
+    Adds Hartree-Fock (HF) and vibrational self-consistent field (VSCF) initial
+    point classes.
+    :class:`~qiskit_nature.initial_points.HFInitialPoint` and
+    :class:`~qiskit_nature.initial_points.VSCFInitialPoint`,
```

Comment:
I do not think this is where these are right - later it has correct references to qiskit_nature.algorithms.initial_points but others here are not correct.